twake/frontend/src/app/services/languages/locale/en.js

```diff
@@ -1972,4 +1972,31 @@ export default {
   'components.richtexteditor.toolbar.ordered-list': 'Ordered list',
   'components.richtexteditor.toolbar.blockquote': 'Blockquote',
   'components.richtexteditor.toolbar.code-block': 'Code Block',
+  'components.locked_features.locked_history_banner.title':
+    'You have just reached the message limit',
+  'components.locked_features.locked_history_banner.description':
+    "Your company has over 10000 messages. That's a lot! Upgrade now to access all your message history.",
```

Comment:
Change it to "Your company has a lot of messages! Upgrade now to access all your message history." (also the french version)

Reply:
@stephanevieira75

Reply:
Done